hockey/indivPlayerFunctions.php

```diff
@@ -0,0 +1,63 @@
+<?php
+function playerInfo($LastName){
+    $info = dbQuery("
+    SELECT *
+    FROM hockey_1617
+    WHERE LastName= :var1",
+    array(
+        "var1"=>$LastName)
+    );
+    return $info->fetch();
+};
+
+function displayPlayerInfo ($name){
+    if($_REQUEST['name']==$name){
+        $result = playerInfo($name);
+        return $result;
+    }
+}
+function playerInfo1516($LastName){
+    $info = dbQuery("
+    SELECT *
+    FROM hockey_1516
+    WHERE LastName= :var1",
+    array(
+        "var1"=>$LastName)
+    );
+    return $info->fetch();
+};
+function displayPlayerInfo1516 ($name){
```

Comment:
Also, it looks like you have some repeated functions (like `displayPlayerInfo` and `displayPlayerInfo1516`). It might be easier to make one single function and take a parameter called something like `$Is1516` which could be true or false.